spec/contracts/placeholder_users/shared_contract_examples.rb

```diff
@@ -33,41 +33,50 @@
 shared_examples_for 'placeholder user contract' do
   let(:placeholder_user_name) { 'UX Designer' }
 
-  it_behaves_like 'contract is valid for active admin users only'
+  context 'when user with global permission' do
```

Comment:
I believe it makes sense to also cover the test case of a normal user without any global permission. It should test that the contract validation fails.

Reply:
That's already covered in the `contract is valid for active admins and invalid for regular users` shared example a bit below in https://github.com/opf/openproject/pull/9000/files#diff-f96e66b7dc90e32788d04451c573b79e112e9e7fc6371f81cc41a73a8351b834R42

I do agree It's a bit confusing with all these shared examples to find out what's happening though.